python/cudf/cudf/core/series.py

```diff
@@ -155,7 +155,7 @@ def values(self):
         if len(self) == 0:
             return cupy.asarray([], dtype=self.dtype)
 
-        return cupy.asarray(self.to_gpu_array())
+        return cupy.asarray(self.data.mem)
```

Comment:
If you're going to do this you need to assert that there's no nulls before hand.